exampleCourse/questions/demo/workspace/jupyterlab/workspace/Workbook.ipynb

```diff
@@ -17,18 +29,25 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "x = np.linspace(0, 2*np.pi, 100)\n",
-    "y = np.sin(x)"
+    "#grade (write your code in this cell and DO NOT DELETE THIS LINE)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The autograder will search for cells that start with a keyword (in this example `#grade`), and only use the contents from this cell for grading. \n",
+    "\n",
+    "This will avoid the autograder tring to execute additional lines of code that include several print statements, plots, and use of libraries that are not installed in PL"
```

Comment:
```suggestion
    "This will avoid the autograder trying to execute additional lines of code that include several print statements, plots, and use of libraries that are not installed in PL."
```